launcher/app.py

```diff
@@ -52,10 +52,13 @@ def __init__(self, root, source):
                                  QtCore.Qt.AlignBottom, QtCore.Qt.black)
 
         terminal.init()
+        app_root = os.path.dirname(__file__).replace('\\', '/')
+        res_path = "file:///{}/res/".format(app_root)
```

Comment:
Is this replicating what is happening here? 

https://github.com/getavalon/launcher/blob/16b5d79af91a01032d365192fd00976e9ccd6d94/launcher/lib.py#L24-L26

Reply:
seems like YES, will fix